src/Orleans/Orleans.csproj

```diff
@@ -74,8 +74,9 @@
   </PropertyGroup>
   <ItemGroup>
     <Reference Include="Microsoft.CSharp" />
-    <Reference Include="Newtonsoft.Json">
-      <HintPath>$(SolutionDir)packages\Newtonsoft.Json.6.0.8\lib\net45\Newtonsoft.Json.dll</HintPath>
+    <Reference Include="Newtonsoft.Json, Version=6.0.0.0, Culture=neutral, PublicKeyToken=30ad4fe6b2a6aeed, processorArchitecture=MSIL">
```

Comment:
@jthelin don't want to derail this PR that much, but the reason the interns had issues with the Orleans-Geo project is specifically because they didn't use NuGet, and were adding the reference outside of the tool. When I was reviewing the code I actually had to add the references through NuGet to make the code compile, because I was having versioning problems too. As soon as I added the references with NuGet itself, everything got fixed. You can see that this was the case if you search for a commit with hash e014d09d304870bc2e5c3761963e728908551828 in that private VSO.